packages/components/CHANGELOG.md

```diff
@@ -13,6 +13,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Porsche Next font supports Vietnamese charset
 
+#### Changed
+
+- `Text Field Wrapper` uses slotted styles from getInitialStyles partial
+- `Initial Styles` partial matches new design language
+
+#### Fixed
+
+- `Text Field Wrapper` browser input date/time icons respect color definition in dark theme
```

Comment:
browser input?

Reply:
I used the description from component-v3-status.md 🙈 I removed the word 'browser' hopefully it sounds better now